### PR TITLE
feat(Builders): Add partner wallets + industry wallets

### PR DESCRIPTION
**Pull request by manuelbarbas**

### CODE REVIEW

File: src/content/docs/builders/tools/wallets/custodial/crossmint.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+<ToolLayout
+    introParagraph="
+Crossmint enables you to create application-specific wallets and provides all necessary blockchain infrastructure for complete use cases, including fetching content, signing transactions, and transferring assets."
+    toolName="Crossmint"
+    toolLink="https://docs.crossmint.com/wallets/custodial-wallets/overview"
+    toolDocsLink="https://docs.crossmint.com/wallets/custodial-wallets/overview"
+>
+
+With Crossmint, you can seamlessly integrate web3 capabilities into your website or app. Create wallets for existing users via API and for new users at sign-up. Additionally, you can build quick, plug-and-play activations with no frontend work required.
```
Comment (TheGreatAxios) [inline src/content/docs/builders/tools/wallets/custodial/crossmint.mdx:17]: Proposal

Crossmint enables you to seamlessly integrate key Web3 functionalty into your application or game.
Create wallets invisibly through their simple APIs for new or existing users. You can use this to quickly enable blockchain without forcing seed phrases and passwords onto users.

File: src/content/docs/builders/tools/wallets/custodial/crossmint.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Implementation Example
+
+<Steps>
+
+1. ##### Get an API key
+
+    Please visit developer console, select the [staging](https://staging.crossmint.com/signin?callbackUrl=/console) or [production](https://www.crossmint.com/signin?callbackUrl=/console) environment and get your API Key.
```
Comment (TheGreatAxios) [inline src/content/docs/builders/tools/wallets/custodial/crossmint.mdx:25]: Make staging -> staging/testnet
Make production -> production/mainnet

File: src/content/docs/builders/tools/wallets/custodial/crossmint.mdx
Comment (TheGreatAxios) [inline src/content/docs/builders/tools/wallets/custodial/crossmint.mdx:1]: Let's make sure it's clear that Crossmint is Nebula only pls

File: src/content/docs/builders/tools/wallets/non-custodial/metakeep.mdx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+<ToolLayout
+    introParagraph="MetaKeep user wallets are seamless non-custodial wallets fully owned by end users. Developers can use MetaKeep APIs and Client SDKs to interact with these wallets, obtaining user consent to perform operations."
+    toolName="Metakeep"
+    toolLink="https://docs.metakeep.xyz/docs/create-a-developer-account"
+    toolDocsLink="https://docs.metakeep.xyz/docs/create-a-developer-account"
+>
+
+MetaKeep provides full self-custody of assets without the need for a hardware wallet. They achieve this with a secure, decentralized virtual environment. Their reliable and cost-effective innovations promote mass adoption, and they offer FDIC-inspired insurance policies to enhance user confidence.
```
Comment (TheGreatAxios) [inline src/content/docs/builders/tools/wallets/non-custodial/metakeep.mdx:16]: MetaKeep offers API based wallets and functionality through abstracted hardware wallets.
This achieve this through the use of industry standard hardware wallets managed automatically by their service. Their reliable and cost-effective innovations promote mass adoption. Additionally, the team offers insurance polices to enhance user confidence.

**Review by TheGreatAxios — CHANGES_REQUESTED** [review]

File: src/content/docs/builders/tools/wallets/custodial/crossmint.mdx
Comment (manuelbarbas) [reply src/content/docs/builders/tools/wallets/custodial/crossmint.mdx:1]: Let me know what you think about this. Not totally sure about it but it was the best idea I got so far
Ideally this or other solution can be used for the rest of the tools. Possible tags could be: SKALE Nebula, Europa, Titan, Calypso and Agnostic  

Also, I'm thinking that ideally I could add this to the <ToolLayout> component

Screenshot 2024-07-24 at 12 29 58

**Review by TheGreatAxios — APPROVED** [review]

**Pull request merged**